config/_default/params.toml

```diff
@@ -87,7 +87,7 @@ smartTOCHideUnfocusedChildren = false
   layoutBackgroundBlur = true # only used when heroStyle equals background
   layoutBackgroundHeaderSpace = false
   showBreadcrumbs = false
-  showSummary = true
+  showSummary = false
```

Comment:
I tracked down the reasons and solutions for gaps in list pages in issue #166.

We can temporarily disable summaries in list pages (as you did in this PR), then fix all issues and re-enable the summaries in a follow-up PR.